src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -18,14 +18,16 @@
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Phone;
 import seedu.address.model.person.Priority;
+import seedu.address.model.person.Remark;
 
 public class JsonAdaptedPersonTest {
     private static final String INVALID_NAME = "R@chel";
     private static final String INVALID_PHONE = "+651234";
+    private static final String INVALID_EMAIL = "example.com";
     private static final String INVALID_ADDRESS = " ";
     private static final String INVALID_BIRTHDAY = "15-09-1995";
     private static final String INVALID_PRIORITY = "important";
-    private static final String INVALID_EMAIL = "example.com";
+    private static final String INVALID_REMARK = "invalid";
```

Comment:
Is there really an invalid remark? Are all kinds of strings acceptable, including empty strings?

Reply:
Good catch! I think this got mixed up with me changing the priority :)